### PR TITLE
Fix the multi-arch building error

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

**Comment by LinuxSuRen** [comment]
/unhold

The CI build status passed. See also [here](https://github.com/LinuxSuRen/ks-jenkins/runs/2783184345?check_suite_focus=true).

File: .github/workflows/build-dev-image.yml [hunk 1]
```diff
@@ -46,15 +46,20 @@ jobs:
...
             --value-set version=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.version'] }}
+
+          # there's no a release for the multi-arch building but https://github.com/jenkinsci/custom-war-packager/releases/tag/v2.0-alpha-6
+          wget https://github.com/LinuxSuRen/custom-war-packager/releases/download/v2.0-alpha-6/custom-war-packager-cli-2.0-alpha-7-20210527.031403-1-jar-with-dependencies.jar
+          mv custom-war-packager-cli-2.0-alpha-7-20210527.031403-1-jar-with-dependencies.jar ~/.jenkins-cli/cwp-cli.jar
           jcli cwp --install-artifacts --config-path formula-arm.yaml --value-set output=push \
             --value-set version=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.version'] }}
       - name: Push Image
         run: |
           version=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.version'] }}
```
Comment (JohnNiang) [inline .github/workflows/build-dev-image.yml:57]: I suggest that you get image version in this way:

```js
 version=${{ fromJSON(steps.meta.outputs.version }}
```

See also [here](https://github.com/docker/metadata-action#outputs).

File: .github/workflows/build-dev-image.yml [hunk 2]
```diff
@@ -46,15 +46,20 @@ jobs:
...
-          docker tag kubespheredev/ks-jenkins:{{.version}}-2.249.1 kubespheredev/ks-jenkins:dev-{{.version}}-2.249.1
-          docker push kubespheredev/ks-jenkins:dev-{{.version}}-2.249.1
+          docker tag kubespheredev/ks-jenkins:$version-2.249.1 kubespheredev/ks-jenkins:dev-$version-2.249.1
+          docker push kubespheredev/ks-jenkins:dev-$version-2.249.1
       - name: Push Image ghcr.io
         run: |
+          version=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.version'] }}
```
Comment (JohnNiang) [inline .github/workflows/build-dev-image.yml:62]: The same as above.

**Review by JohnNiang — CHANGES_REQUESTED** [review]

File: .github/workflows/build-dev-image.yml (hunk 1 above)
Comment (LinuxSuRen) [reply .github/workflows/build-dev-image.yml]: Thanks for your suggestion. Great!

**Pull request merged**